go/filemonitor/filemonitor_darwin.go

```diff
@@ -0,0 +1,120 @@
+// +build darwin
+
+package filemonitor
+
+import "github.com/fsnotify/fsevents"
+
+const flagsWorthReloadingFor = fsevents.ItemRemoved | fsevents.ItemModified | fsevents.ItemRenamed
```

Comment:
There is an `ItemCreated`. I copied this list from here: https://github.com/burke/zeus/blob/ff7c2559dc2bc617a4ea2603f04aacd277f7a5a3/ext/fsevents/main.m#L11-L13

There aren't a lot of people talking about the subtleties of FSEvents but a couple places seem to indicate that you'll get an `kFSEventStreamEventFlagItemCreated` followed by an `kFSEventStreamEventFlagItemModified` if you create a file:
http://stackoverflow.com/questions/15194662/file-system-watcher-for-mac-os-x
http://stackoverflow.com/questions/11123007/check-which-files-have-changed-in-macfsevents-in-perl